cyral/resource_cyral_datalabel.go

```diff
@@ -53,12 +40,44 @@ func resourceDatalabel() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"classification_rule": {
+				Description: "Classification rules are used by the " +
+					"[Automatic Data Map](https://cyral.com/docs/policy/automatic-datamap) feature to automatically map " +
+					"data locations to labels.",
+				Optional: true,
+				Type:     schema.TypeSet,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"rule_type": {
```

Comment:
Please add a validation function.

Reply:
Addressed: [Add validation to classification rule fields](https://github.com/cyralinc/terraform-provider-cyral/pull/436/commits/e029fe765aa9dae8aa866f4ede3b8e3b32bf2402)